int32 iControl_type
char Type_Monitor1
char Type_Monitor2
char Type_Monitor3
char Type_Monitor4
float32 x
float32 y
float32 z 
float32 rot_x
float32 rot_y
float32 rot_z
float32 j1
float32 j2
float32 j3 
float32 j4
float32 j5
float32 j6
